boltdesignsystem/bolt
# DS-796 Remove Ratio Object

## Jira

https://pegadigitalit.atlassian.net/browse/DS-796

## Summary

Ratio object is removed from the `global > styles > 05-objects folder`.

## Details

Referring to the previously [research ticket](https://pegadigitalit.atlassian.net/browse/DS-652) we don't need the ratio object anymore.

## How to test

Pull the branch. Check if there's no ratio object in the Bolt repository and usage of that.

## Release notes

The ratio object is no longer in use. We have the old `ratio-component` and the new `ratio-element`.
